### PR TITLE
Responsive homepage[Closes #41]

### LINKED ISSUE #41: Media query needs to be written so the top header becomes an H1 @ 1200px and the bottom header should decrease to an H4
h1 needs to be rewritten so the top h1 increases as the browser size is increased, the bottom h1 should decrease as the browser size is increased

### PR DESCRIPTION
**Pull request by nodatall**

### CODE REVIEW

**Comment by jamestewartjr** [comment]
good. 👍

**Comment by jason00111** [comment]
The tabbing is off in the file Overlay.js and there is a div (class='overlay-message') that does nothing.

File: src/components/Overlay.js [hunk 1]
```diff
@@ -7,11 +7,20 @@ class Overlay extends Component {
...
         <div className='overlay-img' role='presentation'>
         <div className='overlay-header'>
-          <h1 className='overlay-heading'>LIVE TV DASHBOARD SOFTWARE
-              <div className='overlay-message'>Focus your teams and improve your KPIs</div>
-          </h1>
-          <a href='#' className='flat-button trial-button'>Start your free trial</a>
-          <a href='#' className='flat-button demo-button'>Book a demo</a>
+          <div className='overlay-header-text'>
+            <h1 className='overlay-heading'>LIVE TV DASHBOARD SOFTWARE</h1>
+            <div className='overlay-message'>
```
Comment (jason00111) [inline src/components/Overlay.js]: This is the div that doesn't seem to be doing anything.

File: src/components/Overlay.js [hunk 2]
```diff
@@ -7,11 +7,20 @@ class Overlay extends Component {
       <section className='overlay-container'>
         <div className='overlay-img' role='presentation'>
         <div className='overlay-header'>
```
Comment (jason00111) [inline src/components/Overlay.js]: This is where indentation starts to be funky.

**Comment by jason00111** [comment]
Looks good 👍

**Comment by jrob8577** [comment]
@nodatall There's a few things I would like to see changed in here... Would prefer that this had not been merged, but its work to revert at this point.  
* `HomeNavMenuDropdownButton` should be in its own module
* `HomeNavMenuDropdownButton` - why not `DropdownButton`? Perhaps organized into a file structure that suggests it's for the Home Navigation view only?

Please file issues to address these two things so we can track

**Pull request merged**